server/server.go

```diff
@@ -10,6 +10,8 @@ import (
 	"strconv"
 	"time"
 
+	"github.com/openfga/openfga/pkg/typesystem"
+
```

Comment:
Nit: no space

Reply:
👍 interesting my editor didn't adjust those imports like it usually does 🤷